hr_contract/tests/test_123.py

```diff
@@ -0,0 +1,12 @@
+from odoo.addons.base.maintenance.migrations.testing import UpgradeCase, change_version
+
+
+@change_version("saas~17.4")
+class TestPR123(UpgradeCase):
+    def prepare(self):
+        return True
+
+    def check(self, data):
+        get = self.env["ir.module.module"]._get
+
+        self.assertEqual(get("hr_holiday_contract").state, "installed")
```

Comment:
The test is not running? (missing in `__init__`)
```suggestion
        self.assertEqual(get("hr_holidays_contract").state, "installed")
```

Reply:
No, the error is more stupid than that 🤦